SYSTEM/Cos.md

```diff
@@ -8,8 +8,22 @@ ns: SYSTEM
 float COS(float value);
 ```
 
+```
```

Comment:
Descriptions should not be [codefenced](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md#descriptions) :) Will fix, but a good reminder for the future.